diesel/src/pg/types/mod.rs

```diff
@@ -543,8 +543,22 @@ pub mod sql_types {
     #[diesel(postgres_type(oid = 650, array_oid = 651))]
     pub struct Cidr;
 
-    /// The "char" postgresql type is needed for for e.g. setweight https://www.postgresql.org/docs/current/functions-textsearch.html
-    #[doc(hidden)]
+    /// The [`"char"`] SQL type. This is a PostgreSQL specific type. Used for e.g. [setweight]. [Do not use in user tables].
```

Comment:
Why shouldn't this used in tables? I just tried this and it works.

Reply:
The [linked mailing list](https://www.postgresql.org/message-id/2068524.1659384991%40sss.pgh.pa.us) says you are not supposed to use it.

Reply:
:+1: